lib/widgets/fields/padding_field/padding_types/only_padding.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_quest/widgets/fields/padding_field/padding_text_fields.dart';
+
+class OnlyPadding extends StatefulWidget {
+  final Function(EdgeInsets) onChanged;
+
+  const OnlyPadding({
+    super.key,
+  required this.onChanged,
+  });
+
+  @override
+  State<OnlyPadding> createState() => _OnlyPaddingState();
+}
+
+class _OnlyPaddingState extends State<OnlyPadding> {
+  double top=0.0;
+  double bottom=0.0;
+  double right=0.0;
+  double left=0.0;
+  @override
+  Widget build(BuildContext context) {
+    return Container(
```

Comment:
This is a noticeably big build function. Such functions are hard to understand and do not give an easy idea of what part is doing what. Divide this code into smaller chunks by creating smaller widgets or some variables that hold decisions etc and make the structure more clearer.